public/app/core/services/backend_srv.ts

```diff
@@ -198,13 +198,14 @@ export class BackendSrv implements BackendService {
             }
           }
           process()
-            .catch((e) => {
-              console.log(requestId, 'catch', e);
-            }) // from abort
             .then(() => {
               console.log(requestId, 'complete');
               observer.complete();
-            }); // runs in background
+            }) // runs in background
+            .catch((e) => {
+              console.log(requestId, 'catch', e);
+              observer.error(e);
+            }); // from abort
```

Comment:
I missed this:
https://github.com/grafana/grafana/pull/98691#discussion_r1916004939